python/ray/data/preprocessors/transformer.py

```diff
@@ -7,21 +7,31 @@
 
 
 class PowerTransformer(Preprocessor):
-    """Apply power function to make data more normally distributed.
+    """Apply a `power transform <https://en.wikipedia.org/wiki/Power_transform>`_ to
+    make your data more normally distributed.
```

Comment:
Typically we keep this to 1-line, though I'm not sure what the rule is for hyperlinks 😄

Reply:
The rendered text is less than one line, so I think we should be okay. I'm not aware of any convention that prescribes what to do in this case